source/docs/getting-started/drivers-configuration.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Optional Driver Configuration"
+---
+
+## Vagrant Driver
+
+Sometimes you'd like to add another NIC to your vm. You might have a `forwarded_port` you'd like
+to add also. The following snippet is how.
+
+~~~yaml
+  name: vagrant
+  network:
+  - ["forwarded_port", {guest: 80, host: 8080}]
```

Comment:
Could you include the auto_correct attribute here, so an example of that is shown?  It's a darn shame when a node converge fails and that's the reason. 

`- ["forwarded_port", {guest: 80, host: 8080, auto_correct: true}]`

Thanks